tests/Unit/Evolution/Systems/RadiationTransport/M1Grey/Test_M1HydroCoupling.cpp

```diff
@@ -23,8 +23,20 @@ SPECTRE_TEST_CASE("Unit.RadiationTransport.M1Grey.M1HydroCoupling",
       typename RadiationTransport::M1Grey::ComputeM1HydroCoupling<
```

Comment:
Add the sector in this commit and call `TestHelpers::imex::test_sector`.

Reply:
Would something like `Test_System.cpp` on the below branch suffice?
https://github.com/pajkosmi/spectre/blob/grey_m1/tests/Unit/Evolution/Systems/RadiationTransport/M1Grey/Test_System.cpp

Reply:
Yes, something like that.

Reply:
Test_System.cpp has been added to the branch.